code/modules/reagents/chemistry_reactions/other.dm

```diff
@@ -329,3 +329,14 @@
 	name = "laughter"
 	results = list(/datum/reagent/consumable/laughter = 5)
 	required_reagents = list(/datum/reagent/consumable/sugar = 1, /datum/reagent/consumable/drink/banana = 1)
+
+/datum/chemical_reaction/xenoneuro
+	name = "Toxin"
+	results = list(/datum/reagent/toxin = 1)
+	required_reagents = list(/datum/reagent/toxin/xeno_neurotoxin = 1, /datum/reagent/medicine/spaceacillin = 1)
+
+/datum/chemical_reaction/xenogrow
+	name = "Mindbreaker"
+	results = list(/datum/reagent/toxin/mindbreaker = 1)
```

Comment:
Our current hallucination code is old and awful. We should not rely on it.
We need to port /tg/ hallucinations and tweak it to our game mode.

Reply:
I know, the hallucinations don't do shit.

The only relevant "downsides" are that your health status goes crazy, and sometimes you get a hallucination of a human attacking you by punching.

I don't want full hallucinations, because they're gonna either be obvious and ignored, or they're gonna be an admin nightmare of people constantly FF'ing due to fake xenos always on the screen.

Mindbreaker is only obtainable in normal gameplay by getting larval'd while having spacacillin in your body. The assumption is that you're nested immediately before and after it, and are trading a minute or two of non-larval toxin'd time for a druggy FoV and loss of your health display.

Reply:
Hallucinating a human attacking you is an issue. It goes against our setting.
Please remove the hallucinations until they are fixed.

Reply:
Disagree strongly, but okay.

Reply:
Changed, now causes gradually increasing visual blur which should get preety bad towards the end of a 30u OD. Will test locally.

Reply:
Turns out, visual blur doesn't increase without adding in a reference to cycles in body.
That said, I think the minor blur's fine as it's not really punishing, and xenos are no longer dependent on larval toxin for bursts. 

Working as "intended"

Reply:
Fixed blur after some trial and error.

Working as intended.

Reply:
yeah dont use hallucinations

Reply:
> yeah dont use hallucinations

Pulled them like 10 days ago.

Reply:
Mindbreaker toxin is about hallucinations. Make a custom effect using /tg/ status effects on the chemical you want, on addition, instead.